### DISCUSSION TITLE
[Bug] Not-published State Icon Confusing (Checkbox)

### DISCUSSION THREAD

**Discussion by TroyRobinson** [webstudio-is/webstudio-community#58] [Q&A]
(Overall, amazing revision of Webflow) Just a little UI quirk: should the not-published icon state be a checkbox? Thanks!

![image](https://github.com/webstudio-is/webstudio/assets/24643126/f5b6c243-2bdd-4cd7-8e1a-b7c9c1529384)

**Comment by kof** [comment]
cc @istarkov

**Reply by kof** [reply]
@istarkov seems like either icon or tooltip is wrong
